frontend/src/components/TouchableImage.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text, Image, View,
+  TouchableHighlight,
+  Dimensions,
+} from 'react-native';
+
+function TouchableImage({
+  src, name, onPress, isPressed,
+}) {
+  const windowsWidth = Dimensions.get('window').width;
+  const LABEL_SIZE = Math.round(windowsWidth / 500) * 10;
+
+  return (
+    <TouchableHighlight underlayColor="transparent" onPress={() => onPress(name)} style={{ width: '25%', height: '100%' }}>
```

Comment:
Its better to change height: '100%' to aspectRatio: 1

Reply:
too long line, please break it
```suggestion
    <TouchableHighlight
      underlayColor="transparent"
      onPress={() => onPress(name)}
      style={{ width: '25%', height: '100%' }}
    >
```

Reply:
also based on the task I should see component with some dummy text rendered when you click on the icons.
so if I click on the buildings, it should render a Building component which contains a text: "buildings" or something like that. These components are just really a text now.
sorry if it wasn't clear.